packages/types/src/documents/global/user.ts

```diff
@@ -68,6 +68,16 @@ export interface User extends Document {
   appSort?: string
 }
 
+export interface UserBindings extends Document {
+  firstName?: string
+  lastName?: string
+  email?: string
+  status?: string
+  roleId?: string | undefined | null
```

Comment:
Ideally this should be `roleId?: string | null` - should cover the same typing, although I expect `null` isn't required.